### ISSUE TITLE
[Bug] REI Give/Grab functions not working

### ISSUE THREAD

**Issue by Gamemassa** [shedaniel/RoughlyEnoughItems#1599]
### What happened?

When I click items in the REI window, REI does not add those items to my inventory. Also, when I set it to Grab mode instead, I try to drag items into my inventory and they do not go into the slots I try to put them in. Sometimes when I restart my game the items I had tried to add appear in my inventory.

### What mod loaders are you seeing the problem on?

Fabric 1.20.4

### What do you think this bug is of?

- [X] Visual
- [ ] Recipe Lookup
- [X] Cheat Mode
- [ ] Plugin Integration / JEI Plugin Compatibility
- [ ] Others

### Relevant log output

```shell
Here is the latest.log:
https://gist.github.com/Gamemassa/54a1ad8814b6eb5fd515419876ee4b3d

This bug does not cause any crashes, so there is no crash report to attach.
```

### By submitting this issue, I have included the necessary logs by pasting the contents into the correct location or attaching the file as an upload.

- [X] Yes, and I did not use any paste services other than GitHub Gists.

### By submitting this issue, I have confirmed my REI and REI's dependencies are up to date.

- [X] Yes

**Comment by Gamemassa** [comment]
Update: The Give issue and the Grab issue may be two separate issues. I loaded up a different instance of Fabric 1.20.4 with only REI and its prerequisites, and I set REI cheating mode to "Give" and it works, so I think in my primary instance there may be a conflict causing that (I'll do some debugging and narrow it down). When I set cheating mode to "Grab" and tried dragging items into my inventory, the items still would not be placed in my inventory, so I think that might be an issue within REI.

**Comment by Gamemassa** [comment]
Update: Turns out that updating Fabric API from 0.96.3 to 0.96.11 (the latest one available) fixed the issue with the Grab mode. Still working on the issue with the Give mode.

**Comment by Gamemassa** [comment]
Update: Ok, so the Give issue is definitely a mod conflict (of course). Gonna do some tinkering and try to isolate the conflict.

**Comment by Gamemassa** [comment]
I haven't isolated any conflicts in my modpack yet, but I found another bug that may be related to the Give issue. When I give myself an item using REI, and then destroy it using either REI or the Destroy Item box in Creative, it doesn't appear to be completely destroyed, because if I give myself the same item again, I suddenly have two of that item (and so on and so forth with subsequent attempts).

Update: ~No idea what, but something, just fixed the incomplete-deletion bug I just described. I'm doing a sort of binary test to filter a handful of mods at a time, and in the most recent group, the only mod that even sounds like it could've fixed that is Better Resource Runtime Pack (BRRP), so I assume that fixed it.~ Correction, this issue is still happening.

**Comment by Gamemassa** [comment]
Update: Not only have I isolated the conflict as being Inventorio, but I think the bug is mostly (if not completely) the fault of that mod. When I give myself an item from REI in Creative mode, the item doesn't pop up, but then when I switch to Survival or Adventure mode, the item I had tried to give pops in, and giving other items from REI works fine. I'm going to copy this error report over at the Inventorio GitHub and see what they say.

Update: Rubix (the dev of Inventorio) says this is definitely a bug, and they'll look into it on their end soon when they have time.
